examples/ballot_line_result.xml

```diff
@@ -4,4 +4,5 @@
   <candidate_id>22003</candidate_id>
   <entire_district>no</entire_district>
   <votes>24</votes>
+  <certification>certified</certification>
```

Comment:
`certification` isn't an element. It's just an attribute. You can just remove this here.